index.js

```diff
@@ -78,11 +78,6 @@ xapp.init(xappConfig, (_unused, token) => {
 }) // eslint-disable-line
 
 function bootApp() {
-  if (PRODUCTION_ENV) {
-    app.set("forceSSLOptions", { trustXFPHeader: true }).use(forceSSL)
```

Comment:
I think we might be able to remove the `express-force-ssl` package now.

Reply:
Thanks for pointing this out: https://github.com/artsy/metaphysics/pull/3843